torch/csrc/jit/tracer.h

```diff
@@ -216,6 +216,12 @@ void addInputs(Node *n, const char * name, std::array<bool, N> value) {
 }
 
 inline void ensureUnique(const char * name, const at::Tensor& tensor) {
+  auto& state = getTracingState();
```

Comment:
It might be a good idea to rename this function to `ensureUniqueIfOutOfPlaced` or sth like that. It's not really used outside of auto generated code, but it would clarify its semantics a bit